android/src/main/java/com/gr4vy/embedreactnative/EmbedReactNativeModule.java

```diff
@@ -86,6 +87,7 @@ public static void setOptionalValue(ReadableMap source, WritableMap target, Stri
 
   public EmbedReactNativeModule(ReactApplicationContext context) {
     super(context);
+    reactContext = context;
```

Comment:
So that `Gr4vyActivity` has access to the React context in order to emit the `cardDetailsChanged` event